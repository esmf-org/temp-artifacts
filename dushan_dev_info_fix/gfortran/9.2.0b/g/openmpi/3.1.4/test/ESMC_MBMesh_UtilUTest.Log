2022-07-21 20:17:09
20220721 202642.528 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220721 202642.528 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220721 202642.528 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220721 202642.528 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220721 202642.528 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET0 Running with ESMF Version   : v8.4.0b06-9-g32178349b8
20220721 202642.529 INFO             PET0 ESMF library build date/time: "Jul 21 2022" "20:15:38"
20220721 202642.529 INFO             PET0 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf
20220721 202642.529 INFO             PET0 ESMF_COMM                   : openmpi
20220721 202642.529 INFO             PET0 ESMF_MOAB                   : enabled
20220721 202642.529 INFO             PET0 ESMF_LAPACK                 : enabled
20220721 202642.529 INFO             PET0 ESMF_NETCDF                 : disabled
20220721 202642.529 INFO             PET0 ESMF_PNETCDF                : disabled
20220721 202642.529 INFO             PET0 ESMF_PIO                    : disabled
20220721 202642.529 INFO             PET0 ESMF_YAMLCPP                : enabled
20220721 202642.529 INFO             PET0 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
20220721 202642.529 INFO             PET0 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
20220721 202642.530 ERROR            PET0 /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/src/ESMCI_MBMesh_Glue.C:2507 MBMesh_GetElemCreateInfo() The MOAB library returned an error  - /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/include/ESMCI_MBMesh.h, line:350:Elems not finalized, so num_orig_elem not set.
20220721 202642.530 INFO             PET0 PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
20220721 202642.530 INFO             PET0 PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
20220721 202642.530 INFO             PET0 PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
20220721 202642.530 INFO             PET0 Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
20220721 202642.530 INFO             PET0 Finalizing ESMF
20220721 202642.528 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220721 202642.528 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220721 202642.528 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220721 202642.528 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220721 202642.528 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET1 Running with ESMF Version   : v8.4.0b06-9-g32178349b8
20220721 202642.528 INFO             PET1 ESMF library build date/time: "Jul 21 2022" "20:15:38"
20220721 202642.528 INFO             PET1 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf
20220721 202642.528 INFO             PET1 ESMF_COMM                   : openmpi
20220721 202642.528 INFO             PET1 ESMF_MOAB                   : enabled
20220721 202642.528 INFO             PET1 ESMF_LAPACK                 : enabled
20220721 202642.529 INFO             PET1 ESMF_NETCDF                 : disabled
20220721 202642.529 INFO             PET1 ESMF_PNETCDF                : disabled
20220721 202642.529 INFO             PET1 ESMF_PIO                    : disabled
20220721 202642.529 INFO             PET1 ESMF_YAMLCPP                : enabled
20220721 202642.529 INFO             PET1 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
20220721 202642.529 INFO             PET1 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
20220721 202642.529 ERROR            PET1 /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/src/ESMCI_MBMesh_Glue.C:2507 MBMesh_GetElemCreateInfo() The MOAB library returned an error  - /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/include/ESMCI_MBMesh.h, line:350:Elems not finalized, so num_orig_elem not set.
20220721 202642.529 INFO             PET1 PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
20220721 202642.530 INFO             PET1 PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
20220721 202642.530 INFO             PET1 PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
20220721 202642.530 INFO             PET1 Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
20220721 202642.530 INFO             PET1 Finalizing ESMF
20220721 202642.528 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220721 202642.528 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220721 202642.528 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220721 202642.528 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220721 202642.528 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET2 Running with ESMF Version   : v8.4.0b06-9-g32178349b8
20220721 202642.528 INFO             PET2 ESMF library build date/time: "Jul 21 2022" "20:15:38"
20220721 202642.528 INFO             PET2 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf
20220721 202642.528 INFO             PET2 ESMF_COMM                   : openmpi
20220721 202642.529 INFO             PET2 ESMF_MOAB                   : enabled
20220721 202642.529 INFO             PET2 ESMF_LAPACK                 : enabled
20220721 202642.529 INFO             PET2 ESMF_NETCDF                 : disabled
20220721 202642.529 INFO             PET2 ESMF_PNETCDF                : disabled
20220721 202642.529 INFO             PET2 ESMF_PIO                    : disabled
20220721 202642.529 INFO             PET2 ESMF_YAMLCPP                : enabled
20220721 202642.529 INFO             PET2 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
20220721 202642.529 INFO             PET2 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
20220721 202642.530 ERROR            PET2 /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/src/ESMCI_MBMesh_Glue.C:2507 MBMesh_GetElemCreateInfo() The MOAB library returned an error  - /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/include/ESMCI_MBMesh.h, line:350:Elems not finalized, so num_orig_elem not set.
20220721 202642.530 INFO             PET2 PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
20220721 202642.530 INFO             PET2 PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
20220721 202642.530 INFO             PET2 PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
20220721 202642.530 INFO             PET2 Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
20220721 202642.530 INFO             PET2 Finalizing ESMF
20220721 202642.528 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220721 202642.528 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220721 202642.528 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220721 202642.528 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220721 202642.528 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220721 202642.528 INFO             PET3 Running with ESMF Version   : v8.4.0b06-9-g32178349b8
20220721 202642.528 INFO             PET3 ESMF library build date/time: "Jul 21 2022" "20:15:38"
20220721 202642.528 INFO             PET3 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf
20220721 202642.528 INFO             PET3 ESMF_COMM                   : openmpi
20220721 202642.529 INFO             PET3 ESMF_MOAB                   : enabled
20220721 202642.529 INFO             PET3 ESMF_LAPACK                 : enabled
20220721 202642.529 INFO             PET3 ESMF_NETCDF                 : disabled
20220721 202642.529 INFO             PET3 ESMF_PNETCDF                : disabled
20220721 202642.529 INFO             PET3 ESMF_PIO                    : disabled
20220721 202642.529 INFO             PET3 ESMF_YAMLCPP                : enabled
20220721 202642.529 INFO             PET3 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
20220721 202642.529 INFO             PET3 Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
20220721 202642.529 ERROR            PET3 /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/src/ESMCI_MBMesh_Glue.C:2507 MBMesh_GetElemCreateInfo() The MOAB library returned an error  - /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/gfortran_9.2.0b_openmpi_g_dushan_dev_info_fix/esmf/src/Infrastructure/Mesh/include/ESMCI_MBMesh.h, line:350:Elems not finalized, so num_orig_elem not set.
20220721 202642.529 INFO             PET3 PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
20220721 202642.530 INFO             PET3 PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
20220721 202642.530 INFO             PET3 PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
20220721 202642.530 INFO             PET3 Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
20220721 202642.530 INFO             PET3 Finalizing ESMF
