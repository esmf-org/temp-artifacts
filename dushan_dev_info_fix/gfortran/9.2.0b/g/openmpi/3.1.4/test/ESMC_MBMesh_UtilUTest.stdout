2022-07-21 20:17:09
[1658435202.456049] [h12c09:18383:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.456050] [h12c09:18385:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.456359] [h12c09:18384:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.456465] [h12c09:18386:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.461310] [h12c09:18385:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.461345] [h12c09:18383:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.461927] [h12c09:18384:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1658435202.461999] [h12c09:18386:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131
Beginning Test, file ESMC_MBMesh_UtilUTest.C, line 131NUMBER_OF_PROCESSORS 4
PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
PASS Throw an error from the MBMesh, ESMC_MBMesh_UtilUTest.C, line 160
PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
PASS Test ESMC_MeshSetMOAB, ESMC_MBMesh_UtilUTest.C, line 173
PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
PASS Test ESMC_MeshGetMOAB, ESMC_MBMesh_UtilUTest.C, line 186
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191 PET 2 Test Elapsed Time 59.841 msec.
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191 PET 0 Test Elapsed Time 59.746 msec.
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191 PET 1 Test Elapsed Time 60.451 msec.
Ending Test, file ESMC_MBMesh_UtilUTest.C, line 191 PET 3 Test Elapsed Time 60.452 msec.
